### PR TITLE
Updated the Installation Docs

### PR DESCRIPTION
**Pull request by nshah1729**
Added a command for installing express as it's the most widely used command for installing Express(in all the tutorials I have seen till now! )
Screenshot 2021-11-21 at 2 28 09 PM

### CODE REVIEW

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@nshah1729 That's the same as line 37

**Comment by nshah1729** [comment]
@aravindvnair99 Isn't ```npm install express  --save``` deprecated?
After npm 5.0.0, installed modules are added as a dependency by default, so the --save option is no longer needed

**Comment by aravindvnair99** [comment]
> @aravindvnair99 Isn't `npm install express --save` deprecated? After npm 5.0.0, installed modules are added as a dependency by default, so the --save option is no longer needed

@nshah1729 That is correct. I would be more in favour for that change as opposed to the currently proposed changes.

**Comment by nshah1729** [comment]
@aravindvnair99 So what should I do? 
Should I close this PR and create another PR with ```npm install express --save``` removed and replace it with ```npm i express```??

**Comment by aravindvnair99** [comment]
> @aravindvnair99 So what should I do? Should I close this PR and create another PR with `npm install express --save` removed and replace it with `npm i express`??

@nshah1729 You can edit in same PR itself. Any changes you make in same branch would be reflected here.

As for what to do, my suggestion would be to replace without `--save`.

Note: There are others in the community as well who might have a different opinion. You could either wait for others to pitch in or make the change and wait for others to pitch in. @dougwilson finalises.

**Comment by nshah1729** [comment]
ok Sure!

**Comment by nshah1729** [comment]
I've done!Please check

**Comment by rodion-arr** [comment]
I think this question was raised before and it was decided to leave --save. See https://github.com/expressjs/expressjs.com/pull/936

**Comment by dougwilson** [comment]
With rhe number of times folks keep trying to PR this same change, if merging this will take the PRs stop, then that is probably for the better. There is the note below this that explains the `--save` option and this PR doesn't remove it, so hopefully thay is satisfactory for the readers.

**Review by aravindvnair99 — APPROVED** [review]
As per the official statistics, most downloads are for npm 8, 7, 6 and comparatively much lesser for 1, 2, 3, 4 and 5.  Hence, I'm in favour of this PR especially since this is a starter guide and the chance of someone starting off with an older version of Node.js or npm is unlikely also considering the fact that Node.js 8 and above ships with npm 6+ even though Node.js 12 and 14 are in maintenance now with Node.js 16 being the active LTS.

References:

- npm download stats per version: https://www.npmjs.com/package/npm?activeTab=versions
- npm version shipped with Node.js: https://nodejs.org/en/download/releases/
- Node.js LTS status and release cycle: https://github.com/nodejs/Release#release-schedule

TLDR: I'm in favour of this PR

**Comment by dougwilson** [comment]
Sounds good to me. @nshah1729 can you update this command in the starter guides for the other languages, please?

**Comment by nshah1729** [comment]
@dougwilson Sure! Please merge this PR...I will update the same create PRs in other langs also!

**Comment by dougwilson** [comment]
Just push the updates to the branch this PR is made from and they will be added automatically 👍 no need to make a bunch of PRs if you don't want to. As for merging this PR, either get all the updates added to this PR or get all the PRs made and then we can merge.

**Review by dougwilson — CHANGES_REQUESTED** [review]
Ensure command is consistent across all languages.

File: en/starter/installing.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Enter `app.js`, or whatever you want the name of the main file to be. If you wan
 Now install Express in the `myapp` directory and save it in the dependencies list. For example:
 
 ```sh
```
Comment (aravindvnair99) [inline en/starter/installing.md]:
```suggestion
```console
```

File: ja/starter/installing.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ entry point: (index.js)
 Expressを `myapp` ディレクトリにインストールし、それを依存関係リストに保存してください。例えば：
 
 ```sh
```
Comment (aravindvnair99) [inline ja/starter/installing.md]:
```suggestion
```console
```

File: th/starter/installing.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ entry point: (index.js)
 ต่อไปเป็นการติดตั้ง Express ในไดเรกเทอรี `myapp` และบันทึกไว้ในรายการเกี่ยวโยง (dependencies list) ตัวอย่างเช่น:
 
 ```sh
```
Comment (aravindvnair99) [inline th/starter/installing.md]:
```suggestion
```console
```

File: tr/starter/installing.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ entry point: (index.js)
 
 
 ```sh
```
Comment (aravindvnair99) [inline tr/starter/installing.md]:
```suggestion
```console
```

File: zh-cn/starter/installing.md [hunk 5]
```diff
@@ -40,7 +40,7 @@ entry point: (index.js)
 
 <pre>
 <code class="language-sh" translate="no">
```
Comment (aravindvnair99) [inline zh-cn/starter/installing.md]:
```suggestion
```console
```

File: zh-cn/starter/installing.md [hunk 6]
```diff
@@ -40,7 +40,7 @@ entry point: (index.js)
...
-$ npm install express --save
+$ npm install express
 </code>
 </pre>
```
Comment (aravindvnair99) [inline zh-cn/starter/installing.md]:
```suggestion
```
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
We should  be using `console` instead of `sh` as the former is for terminal interfaces and latter is for shell scripts. Difference:

`sh` / `shell` / `bash`:
```sh
npm i express
```

`console` / `terminal`:

```console
npm i express
```

**Comment by dougwilson** [comment]
Oh neat, I never knew that there was even a console/terminal specifier. Did you verify the jekyll pipeline understands it?

**Comment by aravindvnair99** [comment]
> Oh neat, I never knew that there was even a console/terminal specifier. Did you verify the jekyll pipeline understands it?

@dougwilson Yep, Kramdown supports it. GFM is entirely supported.

Reference:

- https://github.com/expressjs/expressjs.com#formatting
- https://github.com/kramdown/parser-gfm
- https://kramdown.gettalong.org/quickref.html#code-blocks

**Comment by aravindvnair99** [comment]
Hey @nshah1729! Any updates?

**Comment by nshah1729** [comment]
@aravindvnair99 I've committed the changes. Is it done?

File: th/starter/installing.md (hunk 3 above)
Comment (aravindvnair99) [reply th/starter/installing.md]: @nshah1729 you missed this one

**Comment by aravindvnair99** [comment]
> @aravindvnair99 I've committed the changes. Is it done?

@nshah1729 https://github.com/expressjs/expressjs.com/pull/1295#discussion_r754592877

File: th/starter/installing.md (hunk 3 above)
Comment (nshah1729) [reply th/starter/installing.md]: Sorry!

File: zh-cn/starter/installing.md [hunk 7]
```diff
@@ -38,11 +38,8 @@ entry point: (index.js)
 
 在 `myapp` 目录中安装 Express，然后将其保存在依赖项列表中。例如：
 
-<pre>
-<code class="language-sh" translate="no">
-$ npm install express --save
-</code>
-</pre>
+```console
+$ npm install express
```
Comment (aravindvnair99) [inline zh-cn/starter/installing.md:36]: @nshah1729 Can you add `` ``` `` after line 42 here on the next line to close the console formatting? Do leave a blank line before the next heading.

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@nshah1729 Can you bring over changes from #1304, #1308 and the remaining ones into this PR?

**Comment by nshah1729** [comment]
@aravindvnair99 You mean to say that I remove 

```默认情况下，版本为 npm 5.0+ 的 npm install 将模块添加到 `package.json` 文件中的 `dependencies` 列表；对于较早版本的 npm，必须显式指定 `--save` 选项。 今后运行 `app` 目录中的 `npm install` 将自动安装依赖项列表中的模块。``` 

from the current **```PR #1295```** (***the last para***)?

**Review by dougwilson — APPROVED** [review]
